solr/core/src/java/org/apache/solr/core/NodeConfig.java

```diff
@@ -567,6 +575,7 @@ public static class NodeConfigBuilder {
     private String defaultZkHost;
     private Set<Path> allowPaths = Collections.emptySet();
     private List<String> allowUrls = Collections.emptyList();
+    private boolean hideStackTrace = false;
```

Comment:
the default should be the system property value I think. this way it's consistent with the ResponseUtils implementation. (if you only set the system property, both code paths should behave the same)

Reply:
done in latest commit

Reply:
hmm, this doesn't look right :) it will now ignore the builder value. line 616 needs to be 

```
private boolean hideStackTrace = Boolean.getBoolean("solr.hideStackTrace");
```

and line 190 needs to be as before
```
this.hideStackTraces
```

Reply:
Wow!  Thanks.  I don't know what I was thinking ;)